pywps/validator/complexvalidator.py

```diff
@@ -55,11 +55,11 @@ def validategml(data_input, mode):
 
     if mode >= MODE.STRICT:
 
-        from pywps.dependencies import ogr
-        data_source = ogr.Open(data_input.file)
-        if data_source:
-            passed = (data_source.GetDriver().GetName() == "GML")
-        else:
+        try:
+            import fiona
+            data_source = fiona.open(data_input.file)
+            passed = (data_source.driver == "GML")
+        except Exception:
```

Comment:
```suggestion
        except (ModuleNotFoundError, ImportError):
```
We don't want to except `FileNotFoundErrrors` or deprecated API errors